### ISSUE TITLE
Automate collection of contact emails

### ISSUE THREAD

**Issue by ghost** [Ryuno-Ki/cookie-banner-gdpr-request#1]
In most privacy policies (at least the ones that are written properly), the email address of the DPO is mentioned explicitly, and it's often the only email address appearing in the document.

The collection of addresses could hence be automated with some scraping and parsing, which could help populate the list faster.

**Comment by Ryuno-Ki** [comment]
Agreed. But I still would have a human comparing that the correct data was scraped.
Plus, some pages obfuscate all e-mail addresses.

(Apologies for the delay. GitHub burried notifications deep in the "overview").
